test/metabase/events/notification_test.clj

```diff
@@ -0,0 +1,66 @@
+(ns metabase.events.notification-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.events :as events]
+   [metabase.events.notification :as events.notification]
+   [metabase.models.notification :as models.notification]
+   [metabase.notification.core :as notification]
+   [metabase.test :as mt]))
+
+(def supported-topics @#'events.notification/supported-topics)
+
+(defmacro with-temporary-event-topics
+  "Temporarily make `topics` valid event topics."
+  [topics & body]
+  `(let [topics# ~topics]
+     (try
+       (doseq [topic# topics#]
+         (derive topic# :metabase/event))
+       ~@body
+       (finally
+         (doseq [topic# topics#]
+           (underive topic# :metabase/event))))))
+
+(deftest supported-events-with-notification-will-be-sent-test
+  (mt/with-model-cleanup [:model/Notification]
+    (with-temporary-event-topics #{:event/test-notification}
+      (let [topic      :event/test-notification
+            noti-1     (models.notification/create-notification!
```

Comment:
Again, I don't think this is a good abbreviation to introduce. `n1` seems fine in the context, and just `sent` for the accumulator.